MouseMovementTool.cs

```diff
@@ -34,87 +27,71 @@ public Transform cameraRig
                 throw new NotImplementedException();
             }
         }
+        [SerializeField]
+        GameObject m_instructions;
+        [SerializeField]
+        float m_movementMultiplier = .1f;
+        [SerializeField]
+        GameObject m_ringPrefab;
+        [SerializeField]
+        Ring m_ring;
 
-        //public ActionMap actionMap
-        //{
-        //    get
-        //    {
-        //        throw new NotImplementedException();
-        //    }
-        //}
-
-        public bool ignoreLocking
+        void Start()
         {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
-
-        //public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeControl)
-        //{
-        //    throw new NotImplementedException();
-        //}
-
-        public GameObject instructions;
-        public float movementMultiplier = .1f;
-        public GameObject ringPrefab;
-        public Ring ring;
-
-        private void ToggleInstructions()
-        {
-            if(instructions)
-                instructions.SetActive(!instructions.activeSelf);
-        }
-
-        private void Start()
-        {
-            GameObject instance = Instantiate(ringPrefab, cameraRig, false);
+            GameObject instance = Instantiate(m_ringPrefab, cameraRig, false);
             instance.transform.localPosition = new Vector3(0f, -0.09f, 0.18f);
```

Comment:
what is the significance of this position? Should either be a constant or a static readonly value